### PR TITLE
[#1318] Actual 2/2: Added camera availability unit tests

### PR DESCRIPTION
**Pull request by ranbena**
Part of the #1318 fix.

**tl;dr** The actual feature unit tests

NOTICE: This PR depends on #1730, #1731 and #1732 to be merged so defer review till then plz.

#### The #1318 Pull Request stack
1. (#1730) PrepWork 1/3: Added util/poll.js unit tests
2. (#1731) PrepWork 2/3: Made util/poll.js cancellable with cancellablePoll
3. (#1731) PrepWork 3/3: Added cancellablePoll unit tests
4. (#1732) Actual 1/2: Enable “Scan QR” button only if camera available
5. (#1733) Actual 2/2: Added camera availability unit tests  ☜☜☜ _**YOU ARE HERE**_

#### What's this commit about?
Added a test that mocks camera availability, then polls for the expected dom change - "Scan QR" button enable/disable. Had to create an enzyme `waitForElement` utility since the polling is set to a 1s interval.

### CODE REVIEW

File: __tests__/testHelpers.js [hunk 1]
```diff
@@ -27,3 +28,18 @@ export const mockPromiseRejected = (message = 'test error') =>
       reject(new Error(message))
     })
   })
+
+// polls for an enzyme element existence for 2s
+export const waitForElement = async (
+  wrapper,
+  selector,
+  config = { attempts: 10, frequency: 200 }
```
Comment (evgenyboxer) [inline __tests__/testHelpers.js]: On line 32 you stated `frequency` should be `2s`, however its at `200ms`

Comment (ranbena) [reply __tests__/testHelpers.js]: 10 attempts x 200ms frequency = 2 seconds 😁
